llvm/docs/ReleaseNotes.rst

```diff
@@ -95,15 +95,45 @@ Changes to LLVM infrastructure
   `RFC  <https://discourse.llvm.org/t/rfc-ffat-lto-objects-support/63977>`_
   for more details.
 
+* ``ValueTypes.td`` is made richer to represent attributes for
+  ``MVT``.
+
+* ``Intrinsics.td`` is able to emit definitions of
+  ``TypeSig``. ``IntrinsicEmitter`` doesn't depend on
+  ``MachineValueTypes.h`` anymore.
+
+* ``llvm/CodeGen/MachineValueType.h`` is moved from ``llvm/Support``.
```

Comment:
> is moved from

Do you mean _removed_ from? Were the types in ``llvm/CodeGen/MachineValueType.h`` moved to ``ValueTypes.td``?